dialogflow-cx/README.md

```diff
@@ -8,21 +24,21 @@ to create a webhook for a
 
 ### Install dependencies
 
-1. Install pyenv: https://github.com/pyenv/pyenv#installation
+1. Install pyenv: <https://github.com/pyenv/pyenv#installation>
 1. Use pyenv to install
     [the latest version of Python 3](https://www.python.org/downloads/) for
     example, to install Python version 3.10.1, run: `pyenv install 3.10.1`
 1. Create a Python virtual environment with the installed version of Python 3,
     for example, to create a Python 3.10.1 virtual environment called
     `dialogflow-webhook`, run: `pyenv virtualenv 3.10.1 dialogflow-webhook`
-1. Clone this repo and `cd` to the root of the repo
+1. Clone this repository and `cd` to the root of the repository
```

Comment:
Had to cd into `dialogflow-cx` directory in order to pip install requirements.txt